treesource.c

```diff
@@ -173,6 +173,12 @@ static struct marker **add_marker(struct marker **mi,
 	return &nm->next;
 }
 
+void property_add_marker(struct property *prop,
```

Comment:
This belongs in `livetree.c`

Reply:
Is this true with `add_marker` in `treesource.c`?